src/seedu/addressbook/ui/TextUi.java

```diff
@@ -121,48 +99,21 @@ public void showToUser(String... message) {
      * command execution segments.
      */
     public void showResultToUser(CommandResult result) {
-        final Optional<List<? extends ReadOnlyPerson>> resultPersons = result.getRelevantPersons();
-        if (resultPersons.isPresent()) {
-            showPersonListView(resultPersons.get());
-        }
-        showToUser(result.feedbackToUser, DIVIDER);
+        showToUser(format.showResultToUser(result));
```

Comment:
This method behavior is quite different compared to the original?

You can take this activity as a more involved refactoring, i.e. functionalities are maintained but structure of the program changes. Most of other methods you've changed maintain the original behavior quite nicely, which you can use as guide.